incubator/openebs/templates/deployment-openebs-prometheus.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if .Values.policies.monitoring.enabled }}
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "openebs.fullname" . }}-prometheus
+  namespace: {{ .Values.rbac.namespace }}
+  labels:
+    app: {{ template "openebs.name" . }}
+    chart: {{ template "openebs.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    component: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "openebs.name" . }}
+        release: {{ .Release.Name }}
+        component: prometheus
+    spec:
+{{- if .Values.rbac.create }}
```

Comment:
Remove if block and use `{{ template "openebs.serviceAccountName". }}`

Reply:
this file is removed